dto/pagination.go

```diff
@@ -2,56 +2,18 @@ package dto
 
 import "github.com/checkmarble/marble-backend/models"
 
-type PaginationAndSortingInput struct {
-	OffsetId string              `form:"offset_id"`
-	Previous bool                `form:"previous"`
-	Next     bool                `form:"next"`
-	Sorting  models.SortingField `form:"sorting"`
-	Order    models.SortingOrder `form:"order"`
-	Limit    int                 `form:"limit" binding:"max=100"`
+type PaginationAndSorting struct {
```

Comment:
I reworked the DTO and models for pagination, mainly to avoid the endless security alerts from CodeQL (I don't think our implementation had actual security issues, but it definitely made it easier to introduce vulnerabilities accidentally)